### PR TITLE
docs(ecosystem): add fastify-keycloak-adapter

### PR DESCRIPTION
**Pull request by yubinTW**
#### Description

Hi, I created a fastify plugin for working with keycloak server, it could help developer implement the auth mechanism more easily.

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by mcollina** [comment]
The markdown linter is failing, could you take a look?

File: docs/Guides/Ecosystem.md [hunk 1]
```diff
@@ -341,6 +341,8 @@ section.
   authentication for Fastify-based web apps.
 - [`fastify-kafkajs`](https://github.com/kffl/fastify-kafkajs) Fastify plugin
   that adds support for KafkaJS - a modern Apache Kafka client library.
+- [`fastify-keycloak-adapter`](https://github.com/yubinTW/fastify-keycloak-adapter)
```
Comment (jsumners) [inline docs/Guides/Ecosystem.md:344]: Is there any whitespace at the end of this line? We use https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md013---line-length with `strict: false` and `stern: false`. So this line _should_ be okay if there is not any whitespace at the end of the line.

Comment (yubinTW) [reply docs/Guides/Ecosystem.md:344]: It's my fault, I will remove the whitespace.

Comment (yubinTW) [reply docs/Guides/Ecosystem.md]: I have fixed it, please review :)

**Review by RafaelGSS — APPROVED** [review]
LGTM.

**Review by Fdawgs — APPROVED** [review]

**Pull request merged**